eBay-brand-tracker/ebay_bt_venv/Lib/site-packages/_distutils_hack/__init__.py

```
@@ -0,0 +1,239 @@
+
+report_url = (
+    "https://github.com/pypa/setuptools/issues/new?template=distutils-deprecation.yml"
+)
+
+
+def warn_distutils_present():
+    if 'distutils' not in sys.modules:
+        return
+    import warnings
+
+    warnings.warn(
```

Ruff

No explicit `stacklevel` keyword argument found

(B028)